spec/support/contexts/response_tree.rb

```diff
@@ -162,4 +185,13 @@ def expect_read_only_value(path, value)
     el = page.find("[data-path='#{path.join('-')}']")
     expect(el).to have_content(value)
   end
+
+  def temporal_mapping
+    {year: "1i", month: "2i", day: "3i", hour: "4i", minute: "5i", second: "6i"}
+  end
+
+  # Currently works only for paths of depth 1 (children of root)
```

Comment:
Did you try using `data-path` as above? Would be nice to not have this restriction. The `path_selector` logic is also duplicated in a few places above. Can you pull it out and then use it?

Reply:
Another thing that may come in handy is the `$=` CSS selector which matches the end of an attribute. So like `input[id$="_datetime_value_5i"]` would match an input with an id attrib with that ending...